src/compiler/transformers/ts.ts

```diff
@@ -1,4 +1,4 @@
-/// <reference path="../factory.ts" />
+﻿/// <reference path="../factory.ts" />

```

Comment:
What's up with these `^M` line breaks? There are a lot of them in the following files.